Cargo.toml

```diff
@@ -27,7 +27,7 @@ logging_timer = "1.1"
 tempfile = "3.17"
 
 [workspace.dependencies.zip]
-version = "0.6"
+version = "2.2"
```

Comment:
This gets zip with seekable zip entries - used in the installer to eliminate a temp file buffer when extracting from a scie-tote.